internal/cephfs/controllerserver.go

```diff
@@ -311,6 +315,15 @@ func (cs *ControllerServer) CreateVolume(
 			}
 		}
 
+		if !volOptions.BackingSnapshot {
+			// Set metadata on restart of provisioner pod when subvolume exist
+			metadata := k8s.GetVolumeMetadata(req.GetParameters())
```

Comment:
we can move this outside of if block and use it, For now we are calling `k8s.GetVolumeMetadata(req.GetParameters())` twice in createVolume